proposal.md

```diff
@@ -0,0 +1,283 @@
+# Reflect Library Improvements Proposal
+
+## Background
+
+In Eventide's libraries, Reflect has 6 use cases, but 4 of them do not utilize it in the designed way. Instead of making a use of the reflection's actuator, the constant is pulled from the reflection.
+
+The actuator is designed to pass the subject into the method being invoked. Subject could be overriden when needed.
```

Comment:
". The subject can also be overridden, but either way, exactly one argument is always supplied to the protocol method."